docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -275,6 +275,8 @@ Filter and mask environment
 
     These rules do not apply to ``information_schema``.
 
+    ``mask`` can contain conditional expressions such as ``IF`` or ``CASE``, which achieves conditional masking.
```

Comment:
I don't find it should be documented. Mask could be any expression, so functions calls of custom UDFs are possible but we don't document them.

Reply:
I don't have a strong opinion on this. It's just based on my own experience that with more verbose doc the feature can be clearer to the users. Btw Oracle and Snowflake both have this "conditional" concept:
https://docs.oracle.com/database/121/RATUG/GUID-2A303E41-DF8A-4D1E-8AAB-45467D66F082.htm#RATUG4232
https://docs.snowflake.com/en/user-guide/security-column-intro.html#label-security-column-intro-cond-cols